src/App.jsx

```diff
@@ -1,124 +1,103 @@
 import React from 'react';
-import {
-  PayPalScriptProvider,
-  PayPalButtons,
-  FUNDING,
-} from '@paypal/react-paypal-js';
+import { PayPalScriptProvider, PayPalButtons } from '@paypal/react-paypal-js';
 
 function App() {
-  const FUNDING_SOURCES = [
-    FUNDING.PAYPAL,
-    FUNDING.PAYLATER,
-    FUNDING.VENMO,
-    FUNDING.CARD,
-  ];
-
   const initialOptions = {
-    'client-id': 'PAYPAL_CLIENT_ID',
+    'client-id':
```

Comment:
Could you also add the following data attribute to track that this integration comes from integration builder?
```js
'data-sdk-integration-source': 'integrationbuilder_ac'
```